package.json

```diff
@@ -48,6 +48,7 @@
     "test": "./node_modules/.bin/mocha test/unit -R spec -t 5000 --recursive",
     "unit": "./node_modules/.bin/mocha test/unit -R spec -t 5000 --recursive",
     "examples": "node ./examples/DataStax/runner.js",
+    "pre-push": "npx prettier \"{lib,examples,test}/**/*.js\" --write && cargo fmt && cargo clippy --all-targets --all-features -- -D warnings",
```

Comment:
Does this hook git and run those commands before it is pushed to remote?

Reply:
Currently not, but if you think it's recommended to do so, I can add it

Reply:
I don't think it's needed, I'm just asking about the semantics of the `"pre-push"` entry here.